src/components/settings/MergeConfirmationModal.tsx

```
@@ -0,0 +1,133 @@
+interface MergeConfirmationModalProps {
+    secondaryEmail: string;
+    secondaryUserId: string;
+    onClose: () => void;
+}
+
+export const MergeConfirmationModal: React.FC<MergeConfirmationModalProps> = ({
+    secondaryEmail,
+    secondaryUserId,
+    onClose
+}) => {
+    const { isMerging, error, executeMerge } = useAccountMerger();
```

ESLint

[error] Delete `··`

(prettier/prettier)